src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -57,6 +57,7 @@ public static class ScriptConstants
 
         public const string AntaresLogIdHeaderName = "X-ARR-LOG-ID";
         public const string AntaresScaleOutHeaderName = "X-FUNCTION-SCALEOUT";
+        public const string AntaresColdStartHeaderName = "X-MS-COLDSTART";
```

Comment:
Add a test that verifies expected trace message

Reply:
I'll follow up with tests, in the interest of time